File: task1.py
import json
import requests

x = {
  "slackUsername": "Debs",
  "backend": True,
  "age": 22,
  "bio": "My name is Abdulhamid Lawal, i'm a beginner in backend development with little knowledge on python and i hope in gaining tremendous knowledge from HNG9 internship."
}
print(json.dumps(x))

d = requests.get('https://github.com/Debs0007/HNGtask.git', params=x)

File: task1.js
const express = require("express");

const PORT = process.env.PORT || 3500;
const app = express();

app.get("/", (req, res) => {
  const data = {
    slackUsername: "Debs",
    backend: true,
    age: 22,
    bio: "My name is Abdulhamid Lawal, i'm a beginner in backend development with little knowledge on python and i hope in gaining tremendous knowledge from HNG9 internship.",
  };
  res.json(data);
});

app.listen(PORT, () => {
  console.log("server running on " + PORT);
});
